www/apps/docs/content/modules/taxes/backend/create-tax-provider.md

```diff
@@ -127,15 +127,15 @@ The line item tax line object has the following properties:
 
 - `rate`: a number indicating the tax rate.
 - `name`: a string indicating the name of the tax rate.
-- `code`: an optional string indicating the tax code.
+- `code`: a string indicating the tax code.
 - `item_id`: the ID of the line item.
 - `metadata`: an optional object that can hold any necessary additional data to be added to the line item tax lines.
```

Comment:
Can I get you to add below this list the following note:

```suggestion
- `metadata`: an optional object that can hold any necessary additional data to be added to the line item tax lines.

:::note

Tax lines for line item must have a unique `code` and `item_id` combination. Otherwise, the tax lines will be applied multiple times.

:::
```